Java_fibonacci.java

```diff
@@ -0,0 +1,27 @@
+package milestone6;
+
+import java.util.Scanner;
+
+public class JavaFibonacci {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+         Scanner sc=new Scanner(System.in);
+         System.out.println("Enter a number of members in series: ");
+         int n=sc.nextInt();
+         int count=0;
+      
+        BigInteger a=new BigInteger("0");
+        BigInteger b=new BigInteger("1");
+        System.out.print(a+"\n "+b+"\n ");
+        while(count<(n-2))
+         {
+            BigInteger c=new BigInteger(a.add(b).toString());
```

Comment:
Error while executing the code.


$javac milestone6/JavaFibonacci.java
milestone6/JavaFibonacci.java:14: error: cannot find symbol
        BigInteger a=new BigInteger("0");
        ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:14: error: cannot find symbol
        BigInteger a=new BigInteger("0");
                         ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:15: error: cannot find symbol
        BigInteger b=new BigInteger("1");
        ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:15: error: cannot find symbol
        BigInteger b=new BigInteger("1");
                         ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:19: error: cannot find symbol
            BigInteger c=new BigInteger(a.add(b).toString());
            ^
  symbol:   class BigInteger
  location: class JavaFibonacci
milestone6/JavaFibonacci.java:19: error: cannot find symbol
            BigInteger c=new BigInteger(a.add(b).toString());
                             ^
  symbol:   class BigInteger
  location: class JavaFibonacci
6 errors